storage/rocksdb/CMakeLists.txt

```
@@ -98,6 +98,8 @@ ELSE()
       MESSAGE(SEND_ERROR "Missing RocksDB plugin makefile in ${PLUGIN_ROOT}")
     ELSE()
       FILE(READ ${PLUGINMKFILE} PLUGINMK)
+      #Remove line continuation/breaking
+      STRING(REGEX REPLACE "\\\\\n[ \t]+" ""  PLUGINMK ${PLUGINMK})
```

This may not work if there is no space before `\`
For instance, for
```
plugin_SOURCES =\
	file1.cc\
	file2.cc\
	file3.cc
```
we will have `file1.ccfile1.ccfile3.cc`

---

That is correct. 
How about 
```
      STRING(REGEX REPLACE "[ \t]*\\\\\n[ \t]+" " "  PLUGINMK ${PLUGINMK})
```

---

I would add something like
```suggestion
      STRING(REGEX REPLACE "\\\\\n" ""  PLUGINMK ${PLUGINMK})
      STRING(REGEX REPLACE "\\s+" " "  PLUGINMK ${PLUGINMK})
```

---

"\s" does not seem to be a valid escape character in CMake. 
The [CMake specification](https://cmake.org/cmake/help/latest/command/string.html#regex-specification) mentions that `[ \t\r\n]` matches any single whitespace char. 

Something like that?:
```
STRING(REGEX REPLACE "[ \t\r]*\\\\\n[ \t\r]*" " "  PLUGINMK ${PLUGINMK})
```
Where `[ \t\r]*` is zero or more whitespace without linebreaks,
and `\\\\\n` is the "\\" followed by a linebreak,
and `[ \t\r]*` is zero or more whitespace which is the indentation.

---

Yes, @MaisenbacherD
This one should work

---

Changed the regex :)